guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -4,6 +4,7 @@
 Configure an LDAP authentication source to enable users to log in to {Project} with their existing LDAP credentials.
 
 .Prerequisites
+* Your LDAP server is compliant with the link:https://datatracker.ietf.org/doc/html/rfc2307[RFC 2307] schema.
```

Comment:
alternative wording but I have no preference/recommendation:

```suggestion
* Your LDAP server complies with the link:https://datatracker.ietf.org/doc/html/rfc2307[RFC 2307] schema.
```

Reply:
I like this, thanks!